scalafmt-cli/src/main/scala/org/scalafmt/cli/CliArgParser.scala

```diff
@@ -140,7 +140,8 @@ object CliArgParser {
       opt[String]("diff-branch")
         .action((branch, c) => c.copy(mode = Option(DiffFiles(branch))))
         .text(
-          "If set, only format edited files in git diff against provided branch. Has no effect if mode set to `changed`."
+          """If set, only format edited files in git diff against provided branch. Has no effect if mode set to `changed`;
+            |supports both local and remote branches.""".stripMargin
```

Comment:
it doesn't support local or remote branches, it supports any valid git reference -- because it invokes an actual `git` binary. so this clarification is no less inaccurate.

i'd either change `against provided branch` to `against provided valid git reference`, or abandon this commit.